src/reducers/profile-view.js

```diff
@@ -499,6 +500,53 @@ const rightClickedTrack: Reducer<TrackReference | null> = (
   }
 };
 
+const rightClickedCallNodePath: Reducer<RightClickedCallNodePath | null> = (
+  state = null,
+  action
+) => {
+  switch (action.type) {
+    case 'COALESCED_FUNCTIONS_UPDATE': {
+      const { functionsUpdatePerThread } = action;
+
+      if (state === null || !functionsUpdatePerThread[state.threadIndex]) {
+        return state;
```

Comment:
maybe return `null` explicitely here. Especially because of the second part of the condition: if we can't update the value, it's probably better to nullify it.

That said it's probably a bug if we don't have that function update, so I'd suggest to display an error to the console:
```js
if (state === null) {
  return null;
}

const functionUpdate = functionsUpdatePerThread[state.threadIndex];
if (!functionUpdate) {
  console.error(
    `We couldn't find an update function for thread `state.threadIndex`, this is probable a programming mistake.`
  );
  return null;
}

const { oldFuncToNewFuncMap } = functionUpdate;
```